src/index.js

```diff
@@ -0,0 +1,91 @@
+// import _ from 'lodash';
+import './style.css';
+import ToDolist from './toDos.js';
+import complete from './delcomplete.js';
+
+// const removeCompleted = document.querySelector('.removeCompleted');
+
+const myToDolist = new ToDolist();
+
+myToDolist.LoadToDoFromLocal();
+
+function modificarBox(index, valor) {
+  myToDolist.ToDo[index].doneBox = valor;
+  myToDolist.SaveToDolistLocal();
+}
+
+function modifyDescription(index, valor) {
+  myToDolist.ToDo[index].activity = valor;
+  myToDolist.SaveToDolistLocal();
+}
```

Comment:
### great work on your functions, but these two functions are almost the same you can merge them on one function this way 

```javascript

function modify(index, valor,property) {
  myToDolist.ToDo[index][property]= valor;
  myToDolist.SaveToDolistLocal();
}
modify(index,inputTask.value,"activity");
```